common/djangoapps/enrollment/views.py

```diff
@@ -216,21 +225,18 @@ def get(self, request, course_id=None):
             A JSON serialized representation of the course enrollment details.
 
         """
+
         try:
-            return Response(api.get_course_enrollment_details(course_id))
+            return Response(api.get_course_enrollment_details(course_id, request.GET.get('include_expired', '')))
```

Comment:
Since you state in several docstrings that `include_expired` is a Boolean variable, I'd suggest making sure that that is actually the case here. You can use `bool()` to convert the value retrieved for the `include_expired` parameter to a Boolean.